bazel/check_deps/check_non_test_cc_deps.py

```diff
@@ -81,10 +81,10 @@
     # exist. Specially diagnose them to try to provide a more helpful
     # message.
     if repo_base in (
-        "@com_github_google_benchmark",
+        "@google_benchmark",
         "@com_github_protocolbuffers_protobuf",
```

Comment:
Where does this name come from? This is the only mention of it in our codebase.

Reply:
Looks like this was simply incorrect before, replacing.